frontend/src/tests/UsernameInput.spec.ts

```diff
@@ -48,6 +51,69 @@ const idimIdirSearchMock = (isUserFound: boolean): AxiosResponse => {
     }
 };
 
+const idimBceidSearchMock = (
+    isUserFound: boolean,
+    isPermissionError: boolean = false
+): AxiosResponse | AxiosError => {
+    if (isUserFound) {
+        return {
+            data: {
+                found: true,
+                userId: USERID,
+                firstName: FIRSTNAME,
+                lastName: LASTNAME,
+                businessLegalName: BUSINESSLEGALNAME,
+                businessGuid: '', // no need to pass a real guid here
+                guid: TESTUSERGUID, // no need to pass a real guid here
+            },
+            status: 200,
+            statusText: 'Ok',
+            headers: {},
+            config: {
+                headers: {} as AxiosRequestHeaders,
+            },
+        };
+    } else if (isPermissionError) {
+        // todo: investigate how to mock a axios error
```

Comment:
Hi Jota, I added the test for verifying bceid user, but we need to figure out how to mock an axios error, do you mind take some time to see if could find an example? thanks!!

Reply:
Hello, Catherine, just made some changes on the test that was skipped, it is passing now. Can you take a look when you have the time? Thanks!